src/shims/unix/linux/epoll.rs

```diff
@@ -477,4 +485,51 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         }
         Ok(())
     }
+
+    /// Callback function after epoll_wait unblocks
+    fn blocking_epoll_callback(
```

Comment:
Please create a commit where you add this function and use it and then add the blocking logic in a follow-up commit. If you add it right below `epoll_wait`, the diff will be very simple

Reply:
you can turn the last commit back into a staging situation with `git reset --soft HEAD~1`